### DISCUSSION TITLE
Pregunta sobre las personas juridicas, humanas y vulnerables

### DISCUSSION THREAD

**Discussion by ParedesMicaela** [dds-jv/foro#416] [TPA - 2024] [answered]
Buenas tardes! Tenia unas consultas con el tema de las personas vulnerables y el sistema. 
1. Las personas vulnerables deben tener usuario y contraseña o esto es solo para los colaboradores?
2. Las personas vulnerables pueden colaborar o solo pueden hacerlo los colaboradores?
Gracias!

**Comment by flbulgarelli** [comment, marked as answer]
:wave: ¡Hola!

En ambos casos, la especificación es un poco ambigua al respecto, y eso tiene que ver con que en próximas entregas profundizaremos sobre esas cuestiones. Por ahora, por simplicidad y ante la falta de cualquier otro indicio, diría que todas las personas, físicas o jurídicas, independientemente de su rol, puedan registrarse e ingresar al sistema de igual forma. Más adelante veremos si necesitamos proveer otras formas de acceso específicas. 

¡Saludos!

**Reply by flbulgarelli** [reply]
Dicho eso, hay algo que me hace _ruido_ (no de tu pregunta en sí, sino de sus posibles respuestas) a partir de esto: 

> Las personas vulnerables pueden colaborar o solo pueden hacerlo los colaboradores?

Dejo algunas preguntas abiertas. ¿Por qué querríamos que hubiera en el sistema personas que sólo puedan hacer una cosa u otra? Si así fuera, ¿no estaríamos diciendo, implicitamente, que alguien que inicialmente hace donaciones no podría luego recibirlas, o viceversa? A primera vista, esta tipificación parece un tanto estigmatizante, partiendo de la base de que hay personas que estructuralmente siempre asisten y otras que siempre reciben asistencia. ¿No es esta una _situación_ que el sistema del TPA, al menos en su enunciado, busca modificar?

**Comment by ParedesMicaela** [comment]
Es totalmente cierto, ahí ya con el grupo lo pudimos corregir para evitar establecer roles fijos para las personas que brindan ayuda y para los que la reciben. Muchas gracias!
